tools/joystick/README.md

```diff
@@ -0,0 +1,45 @@
+# Joystick
+
+**Hardware needed**: [comma two devkit](https://comma.ai/shop/products/comma-two-devkit), laptop, joystick (optional)
+
+With joystickd, you can connect your laptop to your comma two over the network and debug controls using a joystick or keyboard, however a joystick is recommended for more precise control.
+
+Using a keyboard:
+---
+
+To get started, ssh into your comma two and start joystickd with the following command:
+
+```shell
+tools/joystick/joystickd.py --keyboard
+```
+
+The available buttons and axes will print showing their key mappings. In general, the WASD keys control gas and brakes and steering torque in 5% increments.
+
+Using a joystick:
+---
+
+In order to use a joystick over the network, we need to run joystickd locally from your laptop and have it send `testJoystick` ZMQ packets over the network to the comma two. First connect a compatible joystick to your PC; joystickd uses [inputs](https://pypi.org/project/inputs) which supports many common gamepads and joysticks.
+
+1. Connect your laptop to your comma two's hotspot and open a new ssh shell. Since joystickd is being run on your laptop, we need to write a parameter to let controlsd know to start in joystick debug mode:
+   ```shell
+   # ON YOUR COMMA TWO
+   echo -n "1" > /data/params/d/JoystickDebugMode
```

Comment:
I assume this part is also needed when using a keyboard?

Reply:
A param is written using joystickd.py, but if you use a joystick you have to run it on your laptop so you have to manually write it here